src/ontology/config/mappings-to-uberon-bridge.rules

```diff
@@ -0,0 +1,16 @@
+prefix UBERON:    <http://purl.obolibrary.org/obo/UBERON_>
+prefix CL:    <http://purl.obolibrary.org/obo/CL_>
+
+# Make sure UBERON/CL classes, and only UBERON/CL classes, are on the object side
+subject==UBERON:* -> invert();
+!object==UBERON:* -> stop();
+
+# Ignore any mapping to/from an inexistent or obsolete class.
+predicate==* -> check_object_existence();
```

Comment:
Just for information, the next version of SSSOM-Java will include a newer and (I believe) cleaner syntax to do this — but the existing syntax will remain supported, so nothing to worry about.

Reply:
I will watch out for that!! Thanks!

Reply:
Basically there will be a new filter called `exists()`, allowing to perform an arbitrary action based on whether an entity exists (and is not deprecated) or not in the ontology.

The equivalent of `check_object_existence()` will be:

```
!exists(%{object_id}) -> stop();
```

(Selects all mappings whose object does _not_ — because the filter is negated, `!` – exist in the ontology, and drop them.)

Reply:
Nice! Thanks for letting me know in advance!